contributing/community/review-comments.rst

```diff
@@ -0,0 +1,177 @@
+Respectful Review Comments
+==========================
+
+:doc:`Reviewing issues and pull requests </contributing/community/reviews>`
+is a great way to get started with contributing to the Symfony community.
+Anyone can do it! But before you give a comment, take a step back and think,
+is what you are about to say actually what you intend?
+
+Communicating over the Internet with nothing but text can pose a
+big challenge, especially if you remember that the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+ideas and opinions.
+
+Not everyone speaks English or is able to use a keyboard. Some might
+have Dyslexia or similar conditions that affect their writing.
+
+Not to mention that some might have a bad experience from previous
+contributions (to other projects).
+
+You're not alone in this. This guide will try to help you write
+constructive, respectful and helpful reviews and replies.
+
+.. tip::
+
+    This guide is not about lecturing you to "conform" or give-up
+    your ideas and opinions but helping you to better communicate,
+    prevent possible confusion, and keeping the Symfony community a
+    welcoming place for everyone. **You are free to disagree with
+    someone's opinions, just don't be disrespectful.**
+
+First of, accept that many programming decisions are opinions.
+Discuss trade offs, which you prefer, and reach a resolution quickly.
+It's not about being right or wrong, but using what works.
+
+Tone of voice
+-------------
+
+We don't expect you to be completely formal, or to even write error-free
+English. Just remember one thing: Don't swear, and be respectful to others.
+
+Don't reply in anger or with an aggressive tone. You're angry, we understand
+that, but swearing/cursing and name calling doesn't really encourage anyone to
+help you. Take a deep breath, count to 10 and try to *clearly* explain what problems
+you encounter.
+
+Gender pronouns
+---------------
+
+While not "formally" required it's better to use gender-neutral pronouns.
+Unless someone "indicated" their pronouns, use "they", "them" instead of
+"he", "she", "his", "hers", "his/hers", "he/she", etc.
+
+Try to avoid using wording that may be considered excluding and needlessly gendered,
+like for example words that have a male base. For example we recommend to use other
+words like “folks”, “team”, “everyone” in place of “guys”.
+
+Giving positive feedback
+------------------------
+
+While reviewing issues and pull requests you may run into some suggestions
+(including patches) that don't reflect your ideas, are not good, or downright wrong.
+
+Now, when you prepare your comment, consider the amount of work and time the author
+has spent on their idea and how your response would make them feel;
+
+Did you correctly understand their intention? Or are you making assumptions?
+Which ever you respond, be explicit. Remember people don't always understand your
```

Comment:
"Which ever you respond" is not a grammatically correct/complete sentence. Likely something is intended to the effect of "Whichever way you respond" or "Whatever your response".